packages/fxa-auth-server/docs/swagger/subscriptions-api.ts

```diff
@@ -169,73 +178,123 @@ const OAUTH_MOZILLA_SUBSCRIPTIONS_CUSTOMER_PLAN_ELIGIBILITY = {
 const OAUTH_SUBSCRIPTIONS_IAP_RTDN_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/iap/rtdn',
+  notes: ['Handles a Google Play Real-time Developer Notification.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_CLIENTS_GET = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/clients',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Returns a list of clients and their capabilities.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_INVOICE_PREVIEW_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/invoice/preview',
+  notes: ['Previews an invoice for a new plan.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_INVOICE_PREVIEW_SUBSEQUENT_GET = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/invoice/preview-subsequent',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Preview a list of subsequent invoices.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_COUPON_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/coupon',
+  notes: ['Retrieves coupon details of a valid plan and promotion code.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_PAYMENTMETHOD_FAILED_DETACH_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/paymentmethod/failed/detach',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Detaches a payment method from a customer without any subscriptions.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_PRODUCTNAME_GET = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/productname',
+  notes: ['Returns the product name of a valid product id.'],
 };
 
 const OAUTH_SUBSCRIPTIONS_ACTIVE_SUBSCRIPTIONID_PUT = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/active/{subscriptionId}',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Updates an active subscription for the user.
+    `,
+  ],
 };
 
 const OAUTH_SUBSCRIPTIONS_REACTIVATE_POST = {
   ...TAGS_SUBSCRIPTIONS,
   description: '/oauth/subscriptions/reactivate',
-  notes: ['🔒 Authenticated with OAuth bearer token'],
+  notes: [
+    dedent`
+      🔒 Authenticated with OAuth bearer token
+
+      Reactivate valid customer subscription.
```

Comment:
Stripe and PayPal only; doesn't apply to IAP.

Reply:
Doesn't look like this one was updated? Maybe we didn't hit all of these, but a number of these endpoints won't work for IAP. Since our RPs in some cases hit some of these previously internal APIs directly, it's good to note that for them. We have no way to modify IAP subscriptions in general.